ui/src/styles.scss

```diff
@@ -55,6 +55,10 @@ mat-card {
   font-family: Roboto Regular, "Helvetica Neue", sans-serif;
 }
 
+.app-user-list-card{
+  margin-top: -13.2%;
```

Comment:
some magic margin in percentages, please change. Take a look at styleguide or confirm px margin with Alex